CMSgov/cms-carts-seds
# MDCT-321: Formatting issue with "Last Edited" Column

# Description

Fixes [MDCT-321](https://qmacbis.atlassian.net/browse/MDCT-321)

This updates the Report Items to be more standardized and clean up the last edited time format. 
![Screen Shot 2022-06-28 at 10 23 29 AM](https://user-images.githubusercontent.com/19439679/176203386-a920b3b5-5c44-467d-901f-10d534bad22a.png)


## How to test

- ./dev local
- Login as a state user
- Check the report item
- View/edit the report and certify
- Double check report item is ok
- Repeat the last 4 steps for every possible user type such as Admin or Help Desk

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
